src/compressed_tensors/version.py

```diff
@@ -16,14 +16,17 @@
 Functionality for storing and setting the version info for SparseML
 """
 
-from datetime import date
-
 
 version_base = "0.4.0"
 is_release = False  # change to True to set the generated version as a release version
 
 
-def _generate_version():
+def _generate_version(
+    is_release: bool,
```

Comment:
We don't need to pass these arguments here since they have been already defined in the beginning of this script?

Reply:
It gives an error without it, its not within the scope that the function can recongize when running on pip